### PR TITLE
boards: arduino_nano_33_ble: improve board

### PR DESCRIPTION
**Pull request by gmarull**
The board had numerous issues that had to be addressed (I wonder how a board with such poor quality got merged). Refer to each individual commit for details.

### CODE REVIEW

**Review by carlescufi — DISMISSED** [review]

File: boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts [hunk 1]
```diff
@@ -35,13 +35,18 @@
 			gpios = <&gpio0 6 0>;
 			label = "Blue LED";
 		};
+		led4: led_4 {
+			gpios = <&gpio1 9 0>;
```
Comment (MaureenHelm) [inline boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts:39]: Use `GPIO_ACTIVE_HIGH` or `GPIO_ACTIVE_LOW`

**Review by MaureenHelm — DISMISSED** [review]

File: boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts (hunk 1 above)
Comment (gmarull) [reply boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts]: thanks, added `GPIO_ACTIVE_HIGH`, also adjusted other LEDs (`GPIO_ACTIVE_LOW`).

**Comment by gmarull** [comment]
@carlescufi @MaureenHelm please re-approve :pray:

**Review by MaureenHelm — DISMISSED** [review]

**Comment by gmarull** [comment]
> @carlescufi @MaureenHelm please re-approve :pray:

@carlescufi @MaureenHelm please re-approve :pray: (rebased due to conflicts)

**Review by MaureenHelm — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]
Amen

**Comment by rzr** [comment]
Could this change cause this observed regression:

https://github.com/zephyrproject-rtos/zephyr/issues/49480

**Pull request merged**